cypress/support/session.ts

```diff
@@ -25,25 +26,28 @@ Cypress.Commands.add("setEmailSessionDefaults", (userType: EmailUserTypes) => {
   cy.setCookie(COOKIE_NAME, COOKIE_VALUE)
   cy.setCookie(E2E_USER_TYPE_COOKIE_KEY, userType)
   cy.setCookie(E2E_SITE_KEY, E2E_EMAIL_TEST_SITE.name)
-  cy.setCookie(E2E_COOKIE.Email.key, E2E_EMAIL_ADMIN.email)
+  cy.setCookie(
+    E2E_COOKIE.Email.key,
+    userType === "Email admin" ? E2E_EMAIL_ADMIN.email : E2E_EMAIL_COLLAB.email
+  )
+  cy.setCookie(E2E_COOKIE.Site.key, E2E_COOKIE.Site.value)
 })
 
 Cypress.Commands.add(
   "createEmailUser",
   (
     email: string,
     userType: EmailUserTypes,
-    initialUserType: EmailUserTypes
+    initialUserType: EmailUserTypes,
+    site = ""
```

Comment:
nit: Hmm is this intended? Should we not have a fixture for a default site name?

Reply:
this is the default site name - this is because we want to trigger a way to create a collaborator that's unlinked to any site. 

i could also change this to the e2e email test site (both feel about the same)